pennylane/ops/qubit/matrix_ops.py

```diff
@@ -310,11 +347,39 @@ def compute_decomposition(D, wires):
 
         **Example:**
 
+        # TODO: Update once Identity has been replaced by GlobalPhase
```

Comment:
Nice! Should we add the TODO to the code rather than docs, so users don't see it?